pangeo_forge_runner/dtypes.py

```diff
@@ -0,0 +1,4 @@
+# represents the apache-beam type that
+# Bakery.get_pipeline_options() should return at runtime
+# https://github.com/apache/beam/blob/master/sdks/python/apache_beam/options/pipeline_options.py
+ApacheBeamPipelineOptions = object
```

Comment:
I think object is the top type in python's type system, so this should work for any/all subclasses. A placeholder? A stub for future effort? I'm not sure what it's up to